docs/java/how-to-develop-a-worker-program-in-java.md

```diff
@@ -43,6 +37,10 @@ public class YourWorker {
 }
 ```
 
+After creating the Worker entity, register all Workflow Types and all Activity Types that the Worker can execute.
+For Activities, since they are stateless and thread-safe, instances are used in the registration process instead of the Java Class.
+A Worker can be registered with just Workflows, just Activities, or both.
+
 #### Register Workflow Types
```

Comment:
need to remove this header and move content to "how to register types with a worker in java" or something similar.

Reply:
taken care of